### PR TITLE
io.github.vinegarhq.vinegar

### PR DESCRIPTION
**Pull request by Thelolguy1**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
_allow=devel required for VMProtect to function properly._
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link: https://github.com/vinegarhq/vinegar**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Thelolguy1** [comment]
If it is possible, I would like to push changes from my own organization to the flathub repo. I am unsure if this is possible, but a maintainer mentioned a Github CI for this purpose.

**Comment by Thelolguy1** [comment]
bot, build io.github.vinegarhq.Vinegar

File: io.github.vinegarhq.Vinegar.yml [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+
+  - name: vinegar
+    buildsystem: simple
+    build-commands:
+      - make install GO=$GOROOT/bin/go PREFIX=/app 
+      - install -Dm644 io.github.vinegarhq.Vinegar.metainfo.xml -t /app/share/metainfo
+        # Required for appstream
+      - |
+        icondir=${FLATPAK_DEST}/share/icons/hicolor
+        install -Dm644 catglass.svg ${icondir}/scalable/apps/io.github.vinegarhq.Vinegar.svg
```
Comment (hfiguiere) [inline io.github.vinegarhq.Vinegar.yml:108]: this icon isn't square, appstream will reject it.

Comment (Thelolguy1) [reply io.github.vinegarhq.Vinegar.yml:108]: Inkscape reports it as 36x36

Comment (Thelolguy1) [reply io.github.vinegarhq.Vinegar.yml:108]: Additionally, I believe the buildbot passed the icon check stage

Comment (hfiguiere) [reply io.github.vinegarhq.Vinegar.yml:108]: Must be a github bug then that show it not square.

**Comment by Thelolguy1** [comment]
bot, build io.github.vinegarhq.Vinegar

**Comment by Thelolguy1** [comment]
bot, build io.github.vinegarhq.Vinegar.

**Comment by Thelolguy1** [comment]
bot, build io.github.vinegarhq.Vinegar

File: io.github.vinegarhq.Vinegar.yml (hunk 1 above)
Comment (Thelolguy1) [reply io.github.vinegarhq.Vinegar.yml:108]: That's odd, the flathub bot is now throwing the same error, but `file io.github.vinegarhq.Vinegar.png` reports

`io.github.vinegarhq.Vinegar.png: PNG image data, 128 x 128, 8-bit/color RGBA, non-interlaced`

**Comment by barthalion** [comment]
The build failure usually means some ID mismatch between the icon filename, the desktop and appdata file.

**Comment by Thelolguy1** [comment]
bot, build io.github.vinegarhq.Vinegar

**Comment by Thelolguy1** [comment]
bot, build io.github.vinegarhq.Vinegar

**Comment by Thelolguy1** [comment]
bad build, ignore

**Comment by Thelolguy1** [comment]
bot, build io.github.vinegarhq.Vinegar

**Comment by Thelolguy1** [comment]
Thanks!

**Pull request closed without merging**